### PR TITLE
Update cloud-recording.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5494
Fixed licensing information.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Related: [Teams meeting recording end user documentation](https://aka.ms/recordm
 
 For a Teams user's meetings to be recorded, Microsoft Stream must be enabled for the tenant. In addition, the following prerequisites are required for both the meeting organizer and the person who is initiating the recording:
 
-- User has an Office 365 E1, E3, E5, A1, A3, A5, M365 Business, Business Premium or Business Essentials
+- User has an Office 365 E1, E3, E5, A1, A3, A5, M365 Business Premium, M365 Business Standard and M365 Business Basic.
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:37]:
```suggestion
- User has an Office 365 E1, E3, E5, A1, A3, A5, M365 Business Premium, M365 Business Standard, or M365 Business Basic.
```

Comment (JohanFreelancer9) [reply Teams/cloud-recording.md:37]: Not sure, please confirm that my interpretation is correct.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tonysmit** [comment]
Update: I'm meeting with the PMs to discuss this.

**Pull request merged**